scripts/release.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Prepare for a release
+#
+# All additional options are passed to `rooster release`
+set -eu
+
+export UV_PREVIEW=1
+
+script_root="$(realpath "$(dirname "$0")")"
+project_root="$(dirname "$script_root")"
+
+echo "Updating metadata with rooster..."
+cd "$project_root"
+uv tool run --from 'rooster-blue>=0.0.7' --python 3.12 --isolated -- \
```

Comment:
Doesn't change anything, just a nice alias that clarifies that it's something similar to `pipx`:

```suggestion
uvx --from 'rooster-blue>=0.0.7' --python 3.12 --isolated -- \
```